docs/dotnet-configuration.md

```diff
@@ -35,7 +122,7 @@ By default, Scope .NET agent adds both `platform.name` and `dotnet.runtime` meta
 ### Using Environment Variables
 
 ```sh
-export SCOPE_CONFIGURATION="sample.key1,sample.key2"
+SCOPE_CONFIGURATION="sample.key1,sample.key2"
```

Comment:
Please check in your agents if the " character is trimmed. In the Java agent, I had to fix this issue.